sdk/deploy.js

```diff
@@ -41,7 +37,7 @@ const deployCode = async function deployCode(deployURLBase, apiKey, tarPath) {
         if (uploadErr) {
           reject(uploadErr);
         } else if (uploadResponse.statusCode !== 200) {
-          reject(new HTTPError(uploadResponse));
+          reject(new APIError(translateErrorCode(uploadResponse.body.errorCode)));
```

Comment:
Can we call into `handleError.js` logic here to have similar `errorText` handling?

Reply:
Sure I'll break down the handleError logic and call the relevant part.